### PR TITLE
Create Release Notes Page

### PR DESCRIPTION
**Pull request by codemonkey9000**
Added a new release notes page and linked to it from the footer on each page and the homepage. The plan for this page is to be manually updated from the internal release notes page that Shane provides internally.

@srainier please make sure that all of the current notes I have taken from the internal notes are relevant to an external audience and able to be shared.

PR Review Checklist:
- [x] Test search functionality on desktop
- [x] Test search functionality on mobile
- [x] Test filter functionality
- [x] Test mobile header navigation
- [x] Test desktop header navigation
- [x] Test desktop homepage navigation
- [ ] Run `npm run check-links` with 0 broken links
- [ ] Run `npm run write-good <files being edited>` to "lint" the docs

### CODE REVIEW

File: source/includes/_release-notes.html.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Product Release Notes
+
+These release notes combine releases for the web (BRAIN Dashboard), the backend services (Bonsai AI Engine), and Bonsai's API. All releases are tagged by date for the web, backend, and API. Releases are based on version for the SDKs and CLI which are available on GitHub.
```
Comment (srainier) [inline source/includes/_release-notes.html.md]: This line "Releases are based on version for the SDKs and CLI which are available on GitHub." This had to be explained to me. What was confusing is that I read 'Releases' as being Platform releases, but it's referring to SDK and CLI versions. I would suggest instead: "SDK and CLI are released separately and have different version numbers. They are available on GitHub."

File: source/includes/_release-notes.html.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Release Notes for CLI: [https://github.com/BonsaiAI/bonsai-cli/blob/master/CHANGELOG.md]()
+
+Release Notes for Python SDK (bonsai-ai): [https://github.com/BonsaiAI/bonsai-sdk/blob/master/bonsai-ai/CHANGELOG.md]()
+
+### 2018.02.23
+
+Various bug fixes, including:
+
+* Fix for concept threshold termination tags not terminating as expected
```
Comment (srainier) [inline source/includes/_release-notes.html.md]: I think this is an undocumented (publicly) experimental tag.

File: source/includes/_release-notes.html.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Product Release Notes
+
+These release notes combine releases for the web (BRAIN Dashboard), the backend services (Bonsai AI Engine), and Bonsai's API. All releases are tagged by date for the web, backend, and API. Releases are based on version for the SDKs and CLI which are available on GitHub.
+
+Release Notes for CLI: [https://github.com/BonsaiAI/bonsai-cli/blob/master/CHANGELOG.md]()
+
+Release Notes for Python SDK (bonsai-ai): [https://github.com/BonsaiAI/bonsai-sdk/blob/master/bonsai-ai/CHANGELOG.md]()
+
```
Comment (srainier) [inline source/includes/_release-notes.html.md]: As mentioned, there was a `2018.03.07` release, which notably had a number of bug fixes for resume training.

File: source/includes/_release-notes.html.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+* Fix for concept threshold termination tags not terminating as expected
+* Removed limitation for Inkling variables ending in underscore followed by a number
+* Fixed backend race conditions causing dropped simulators
+
+### 2018.02.13
+
+Notable features:
+
+* You can now Resume Training a BRAIN or Restart Training instead
+* Tags for reward and iteration based termination of concepts
```
Comment (srainier) [inline source/includes/_release-notes.html.md]: Also undocumented experimental tag.

**Comment by codemonkey9000** [comment]
Thank you Shane! Updates have been made, please approve if this looks good for now and we can make further updates next release.

**Pull request merged**